pkgs/development/python-modules/esper/default.nix

```diff
@@ -0,0 +1,30 @@
+{
+  buildPythonPackage,
+  fetchFromGitHub,
+  flit-core,
+  lib,
+}:
+
+buildPythonPackage rec {
+  pname = "esper";
+  version = "3.2";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "benmoran56";
+    repo = "esper";
+    rev = "refs/tag/v${version}";
```

Comment:
See https://github.com/NixOS/nixpkgs/pull/311759#discussion_r1601107636

Reply:
I set `refs/tag/v${version}`, anything missing?

Reply:
tag -> tags

Reply:
one day i'll finally not do stupid mistakes :sweat_smile: